Cargo.toml

```diff
@@ -16,12 +18,8 @@ appveyor  = { repository = "Gilnaa/globwalk" }
 [dependencies]
 walkdir = "2"
 ignore = "0.4.11"
-bitflags = "1.2"
+bitflags = "2"
 
 [dev-dependencies]
 tempfile = "3"
 docmatic = "0.1.2"
-
-# We're not using backtrace, really. It is just a deep dependency of docmatic.
-# Locking down its version so that we can compile the tests using rustc 1.31.0
```

Comment:
It didn't seem to compile on 1.31.0 anyway, so this pinning seemed unnecessary and I removed it